pkgs/dart_mcp_server/lib/src/mixins/dtd.dart

```diff
@@ -958,3 +960,52 @@ extension type DebugSession.fromJson(Map<String, Object?> _value)
     if (vmServiceUri != null) 'vmServiceUri': vmServiceUri,
   });
 }
+
+@visibleForTesting
+/// Manages a log of errors with a maximum size in terms of total characters.
+class ErrorLog {
```

Comment:
[nit] https://dart.dev/effective-dart/documentation#do-put-doc-comments-before-metadata-annotations

```suggestion

/// Manages a log of errors with a maximum size in terms of total characters.
@visibleForTesting
class ErrorLog {
```